scripts/16_set-deposit-limit.js

```diff
@@ -0,0 +1,23 @@
+const {
+    callTokenManagerFunction,
+} = require('./helpers/10-call-token-manager-fn');
+
+async function setDepositLimit(minOrMax, newLimit) {
+    let functionToCall;
+    if (minOrMax === 'min') {
```

Comment:
I know this looks better, but it can be simplified with a ternary:
```ts
const functionToCall = minOrMax === 'min' ? 'setMinimumDeposit' : 'setMaximumDeposit'
```

Reply:
Or even:
```ts
await callTokenManagerFunction(minOrMax === 'min' ? 'setMinimumDeposit' : 'setMaximumDeposit', [newLimit]);
```

Reply:
What if I send 'anything'? Do i want it to do setMaximumDeposit?

Reply:
You are absolutely right.

Reply:
Or
```js
if !['min', 'max'].includes(minOrMax) { throw error };
[Ternary]
await callTokenManagerFunction(...)
```